docs/samples/client/oauth/csharp/UnbrandedCredential.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// Demonstrates creating a client with an unbranded authentication token provider.
+using System;
+using System.ClientModel;
+using System.ClientModel.Primitives;
+using System.Collections.Generic;
+
+namespace UnbrandedTypeSpec
+{
+    /// <summary>
+    /// Demonstrates creating a client with an unbranded authentication token provider.
+    /// </summary>
+    public partial class UnbrandedTypeSpecClient
```

Comment:
This client needs at least one method that calls an endpoint also.

Reply:
I added SayBye and SayByeRestClient methods that use the endpoint